examples/VIAME_OcclusionSaliency.ipynb

```diff
@@ -0,0 +1,504 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[('LUTJANUSCAMPECHANUS-170151107',\n",
+       "  {'stats': OrderedDict([('mean',\n",
+       "                 array([298.54468, 197.1837 ], dtype=float32)),\n",
+       "                ('std', array([217.16107, 119.08545], dtype=float32)),\n",
+       "                ('min', array([19., 22.], dtype=float32)),\n",
+       "                ('max', array([1155.,  599.], dtype=float32)),\n",
+       "                ('shape', (1399, 2))])}),\n",
+       " ('SERRANUSANNULARIS-170024201',\n",
+       "  {'stats': OrderedDict([('mean',\n",
+       "                 array([38.727272, 26.630302], dtype=float32)),\n",
+       "                ('std', array([7.9601765, 5.607381 ], dtype=float32)),\n",
+       "                ('min', array([22., 13.], dtype=float32)),\n",
+       "                ('max', array([58., 47.], dtype=float32)),\n",
+       "                ('shape', (825, 2))])}),\n",
+       " ('MALACANTHUSPLUMIERI-170070301',\n",
+       "  {'stats': OrderedDict([('mean',\n",
+       "                 array([68.26818 , 32.392426], dtype=float32)),\n",
+       "                ('std', array([36.12234 , 18.063078], dtype=float32)),\n",
+       "                ('min', array([17., 13.], dtype=float32)),\n",
+       "                ('max', array([147.,  90.], dtype=float32)),\n",
+       "                ('shape', (660, 2))])}),\n",
+       " ('BACKGROUND',\n",
+       "  {'stats': OrderedDict([('mean',\n",
+       "                 array([100.67889,  75.62478], dtype=float32)),\n",
+       "                ('std', array([82.251465, 46.612217], dtype=float32)),\n",
+       "                ('min', array([17., 19.], dtype=float32)),\n",
+       "                ('max', array([472., 220.], dtype=float32)),\n",
+       "                ('shape', (573, 2))])}),\n",
+       " ('EPINEPHELUSMORIO-170021211',\n",
+       "  {'stats': OrderedDict([('mean',\n",
+       "                 array([166.65672 ,  96.779854], dtype=float32)),\n",
+       "                ('std', array([88.33692, 53.01668], dtype=float32)),\n",
+       "                ('min', array([23., 18.], dtype=float32)),\n",
+       "                ('max', array([592., 307.], dtype=float32)),\n",
+       "                ('shape', (536, 2))])})]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Sort annotations by frequency in dataset (top-5)\n",
+    "sorted(dset.boxsize_stats()['perclass'].items(), key=lambda x: x[1]['stats']['shape'][0], reverse=True)[:5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The test image\n",
+    "We will test this application on the following image.\n",
+    "We know that this image contains the ImageNet classes of \"boxer\" and \"tiger cat\"."
```

Comment:
We should update this narrative component to discuss the new image, like what it is called and which of our two classes picked out in out blackbox formulation this is the true class of this image. If we're considering this one image maybe we just want the one, true class back from the blackbox?

Reply:
Good catch- I think I'm going to go with the ground-truth class and the predicted class (and try to find an image where these are not the same). Thanks~